modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -308,8 +308,10 @@
   "selectedMenuItem": "✓ {{ label }}",
   "selectAll": "Select All",
   "selectBoxMessage": "{{ num }} {{ label }} on this page selected.",
```

Comment:
I just noticed that a user can select all on one page, switch to the next page and select all there (one at a time) then this reads something like "20 articles on this page are selected" even if there are only 10 on the page. That fix can be a separate PR.

Reply:
Well seen, I fixed it.